L.SimpleGraticule.js

```diff
@@ -116,15 +117,15 @@ L.SimpleGraticule = L.LayerGroup.extend({
     },
 
     buildXLine: function(x) {
-        var bottomLL = new L.LatLng(this._bounds.getSouth(), x);
-        var topLL = new L.LatLng(this._bounds.getNorth(), x);
+        var bottomLL = new L.LatLng(-180, x);
+        var topLL = new L.LatLng(180, x);
 
         return new L.Polyline([bottomLL, topLL], this.lineStyle);
     },
 
     buildYLine: function(y) {
-        var leftLL = new L.LatLng(y, this._bounds.getWest());
-        var rightLL = new L.LatLng(y, this._bounds.getEast());
+        var leftLL = new L.LatLng(y, -360);
```

Comment:
My motivation was using degrees outside of this are meaningless in some contexts.

I had considered making them optionally cut off, or repeat, or continue as you had it.  The only problem with repeating is if you use an interval that does not mod 360 to 0, the repeating is off, so I just cut it off at 360.  I probably should have filed an issue ahead of the pr for a discussion...